llvm/lib/IR/Function.cpp

```diff
@@ -387,6 +388,25 @@ Function *Function::Create(FunctionType *Ty, LinkageTypes Linkage,
   return Create(Ty, Linkage, M.getDataLayout().getProgramAddressSpace(), N, &M);
 }
 
+StringRef Function::getDefaultTargetFeatures(const StringRef TargetABI) {
+  Triple T(getParent()->getTargetTriple());
+  StringRef attr = "";
+  if (T.isRISCV64()) {
+    if (TargetABI.contains("lp64d"))
```

Comment:
I don't like this contains for the risc-v abi checks, it should just be equality (maybe case-insensitive) - I'm not sure we have any special prefixes or suffixes that can be used, or will be used by llvm.